mirrord/intproxy/src/agent_conn.rs

```diff
@@ -46,6 +46,7 @@ pub enum AgentConnectionError {
 /// Directive for the proxy on how to connect to the agent.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub enum AgentConnectInfo {
+    ExtProxy(SocketAddr),
```

Comment:
Considering how we use `Ext` to extend types (with the trait extensions), maybe go for long name here instead of just `ExtProxy`.